deploy.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+if ([ $TRAVIS_PULL_REQUEST == "false" ])
```

Comment:
The BitFlux deploy script has a few extra checks - e.g. not a fork.

Do we want this too?